jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```diff
@@ -119,11 +119,24 @@
 
     - name: Clone the metal3-dev-env repo
       git:
-        repo: "{{ metal3_dev_env_repo }}"
+        repo: "https://github.com/metal3-io/metal3-dev-env.git"
         dest: "/home/{{ ansible_user_id }}/{{ metal3_dir }}"
-        version: "{{ metal3_dev_env_branch }}"
+        version: "main"
```

Comment:
`metal3_dev_env_repo` and `metal3_dev_env_branch` were introduced to differentiate bml runs from this repo and m3-dev-env repo, so that if it the latter it will run the tests with updated branch. Now, what is the motivation behind this change?